### DISCUSSION TITLE
在 PubMed 网站通过 zotero connector添加的文献条目，会附加一个PubMed entry。有没有快捷方式删除条目下面的PubMed entry，

### DISCUSSION THREAD

**Discussion by yuzuan** [windingwind/zotero-actions-tags#237] [Q&A]
在 PubMed 网站通过 zotero connector添加的文献条目，会附加一个PubMed entry。有没有快捷方式删除条目下面的PubMed entry。
<img width="139" alt="image" src="https://github.com/windingwind/zotero-actions-tags/assets/54974291/044f787d-1507-410c-8df3-8eaffe1b9c26">

**Comment by l0o0** [comment]
@yuzuan 保存的时候，可以不保存网页快照，在设置 ，常规里，有一个snapshot 网页快照的选项
